code/datums/disease.dm

```diff
@@ -173,6 +174,21 @@ var/list/diseases = typesof(/datum/disease) - /datum/disease
 		var/mob/living/carbon/human/H = affected_mob
 		H.med_hud_set_status()
 
+/datum/disease/proc/get_infection_chance(mob/living/carbon/human/victim)
+	if(ishuman(victim))
+		var/protection = 5 //Our current credit protection on how good are we protected from any sort of disease. Five acts as base protection.
```

Comment:
Why start protection at 5 if you will decrease it by 5 in the end?

Reply:
I clamp it anyways, you are right